### PR TITLE
Fixed YOLOV5 error with Mac

### PR DESCRIPTION
**Pull request by AnishaUdayakumar**
Add fixes to #599

### CODE REVIEW

**Comment by ryanloney** [comment]
@AnishaUdayakumar the notebook is failing CI. Here is what happens when I try to run it on my Windows laptop: 
image

File: notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb [hunk 1]
```diff
@@ -80,7 +115,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sys.path.append(\"./yolov5/\")\n",
+    "sys.path.append(\"220-yolov5-accuracy-check-and-quantization/yolov5\")\n",
```
Comment (adrianboguszewski) [inline notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb:118]:
```suggestion
    "sys.path.append(\"./yolov5\")\n",
```

File: notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb [hunk 2]
```diff
@@ -54,15 +56,48 @@
...
+    "print(\"Convert ONNX model to OpenVINO IR:\")\n",
+    "onnx_path=\"./yolov5/yolov5m/yolov5m.onnx\"\n",
+    "IMAGE_HEIGHT=640\n",
+    "IMAGE_WIDTH=640\n",
+    "model_output_path=\"./yolov5/yolov5m/yolov5m_openvino_model/\"\n",
+    "\n",
+    "# Construct the command for Model Optimizer.\n",
+    "command_mo= f\"\"\"mo\n",
+    "                 --input_model \"{onnx_path}\"\n",
+    "                 --input_shape \"[1,3, {IMAGE_HEIGHT}, {IMAGE_WIDTH}]\"\n",
```
Comment (adrianboguszewski) [inline notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb:93]:
```suggestion
    "                 --input_shape \"[1, 3, {IMAGE_HEIGHT}, {IMAGE_WIDTH}]\"\n",
```

**Review by adrianboguszewski — APPROVED** [review]

**Review by raymondlo84 — COMMENTED** [review]
Please update a few of the print lines (remove the duplicates).

**Comment by AnishaUdayakumar** [comment]
> @AnishaUdayakumar the notebook is failing CI. Here is what happens when I try to run it on my Windows laptop: image

@ryanloney Could you try creating a new virtual environment and test it again, please? I'm unable to replicate this error.

**Comment by AnishaUdayakumar** [comment]
> Please update a few of the print lines (remove the duplicates).

@raymondlo84 I have updated the print lines. Request to please review the messages.

**Review by ryanloney — APPROVED** [review]

**Pull request merged**